File: models/implementations/statistics.py
"""History statistics: explicit architecture using shared differentiable primitives."""
from ._neural import NeuralModel,train_design
MODEL_ID='statistics'

class Model(NeuralModel):
    architecture={'id': 'statistics', 'label': 'History statistics', 'hidden': 0, 'memory': 'none', 'readout': 'none', 'layers': 0, 'description': 'Learned categorical predictors using account and pair statistics; no neural memory or neighborhood aggregation.'}

def train(args,data,labeled=None):
    return train_design(args,data,MODEL_ID,labeled)

File: models/implementations/statistics.js
/* History statistics. See matching Python module for training. */
(function(global){
'use strict';
const registry=typeof module!=='undefined'?require('../../shared/runtime/model-registry'):global.FraudAdapters;
const base=typeof module!=='undefined'?require('./_neural'):global.FraudNeuralPrimitives;
// Architecture: none memory; none readout; 0 graph layers.
const adapter={initialize:base.initialize,update:base.update,readout:base.readout,predict:base.predict};
registry.register('statistics',adapter);
if(typeof module!=='undefined')module.exports=adapter;
})(globalThis);
